debug/testlib.py

```diff
@@ -1070,6 +1070,17 @@ def parkOtherHarts(self):
 
         self.gdb.select_hart(self.hart)
 
+    def disable_pmp(self):
+        # Disable physical memory protection by allowing U mode access to all
+        # memory.
+        try:
+            self.gdb.p("$pmpcfg0=0xf")  # TOR, R, W, X
```

Comment:
Ah, so top-of-range in the pmp0cfg slot implicitly treats bottom of range as address 0x0.  Hadn't noticed that before, but makes sense under the principle of least surprise.